eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Utxo.hs

```diff
@@ -407,10 +534,25 @@ allegraToConwayUtxoPredFailure ::
 allegraToConwayUtxoPredFailure = \case
   Allegra.BadInputsUTxO x -> BadInputsUTxO x
   Allegra.OutsideValidityIntervalUTxO vi slotNo -> OutsideValidityIntervalUTxO vi slotNo
-  Allegra.MaxTxSizeUTxO x y -> MaxTxSizeUTxO x y
+  Allegra.MaxTxSizeUTxO supplied expected ->
+    MaxTxSizeUTxO
+      Mismatch
+        { mismatchSupplied = supplied
+        , mismatchExpected = expected
+        }
   Allegra.InputSetEmptyUTxO -> InputSetEmptyUTxO
-  Allegra.FeeTooSmallUTxO c1 c2 -> FeeTooSmallUTxO c1 c2
-  Allegra.ValueNotConservedUTxO vc vp -> ValueNotConservedUTxO vc vp
+  Allegra.FeeTooSmallUTxO minFee suppliedFee ->
+    FeeTooSmallUTxO
+      Mismatch
```

Comment:
Maybe have a comment about reversal here?

Reply:
The comment would soon be unnecessary due to the follow up PR that will change `allegra` to also use `Mismatch`.